common/app/common/pages.cljc

```diff
@@ -833,6 +873,22 @@
   [data {:keys [id]}]
   (d/dissoc-in data [:components id]))
 
+;; -- Typography
+
+(defmethod process-change :add-typography
+  [data {:keys [typography]}]
+  (update data :typography assoc (:id typography) typography))
```

Comment:
Wouldn't be better that the name of the collection of tipography objects be :tipographies ?

Reply:
I prefer a singular word because the other one is too long

Reply:
eh.. but this is inconsistent with other assets, that are called "pages", "colors" and "components". And may cause confusion, perhaps we don't know if a variable points to the whole collection of typographies or to one particular one; or else we need to name a variable "tipographies" that refers to the "tipography" element. And it's only two characters longer...